package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@curriculumassociates/createjs-accessibility",
-  "version": "1.0.7",
+  "version": "1.0.8",
```

Comment:
From a semver perspective, this should be a minor bump (feature enhancement), not a patch bump.